File: packages/shared/src/types/ad.ts
export interface AdDto {
    id: string;
    imagePath: string;
    redirectUrl: string;
    isActive: boolean;
    createdAt: string;
}

export interface AdCreateRequest {
    redirectUrl: string;
    isActive: boolean;
}

export interface AdUpdateRequest {
    redirectUrl: string;
    isActive: boolean;
}

File: backend/PADIDI.API/Models/Ad.cs
namespace PADIDI.API.Models;

public class Ad
{
    public Guid Id { get; set; }
    public string ImagePath { get; set; } = string.Empty;
    public string RedirectUrl { get; set; } = string.Empty;
    public bool IsActive { get; set; } = true;
    public DateTimeOffset CreatedAt { get; set; }
    public DateTimeOffset UpdatedAt { get; set; }
}
